edxlearndot/management/commands/update_learndot_enrolments.py

```diff
@@ -70,7 +83,7 @@ def handle(self, *args, **options):
                 log.error("No course mappings were found for your specified course IDs.")
             else:
                 log.error("No course mappings were found.")
-            sys.exit(1)
+                sys.exit(1)
```

Comment:
This change looks wrong -- surely we need to exit if either error is encountered above?